library/src/test/java/com/datatorrent/lib/formatter/JsonFormatterTest.java

```diff
@@ -198,15 +199,25 @@ public void testJSONToPOJONoFieldPOJO()
   @Test
   public void testApplication() throws IOException, Exception
   {
-    try {
-      LocalMode lma = LocalMode.newInstance();
-      Configuration conf = new Configuration(false);
-      lma.prepareDAG(new JsonFormatterApplication(), conf);
-      LocalMode.Controller lc = lma.getController();
-      lc.run(2000);// runs for 2 seconds and quits
-    } catch (ConstraintViolationException e) {
-      Assert.fail("constraint violations: " + e.getConstraintViolations());
-    }
+    LocalMode lma = LocalMode.newInstance();
+    Configuration conf = new Configuration(false);
```

Comment:
What is the purpose of this code? This test case should cover JsonFormatter. It looks like it is testing generic engine behavior instead?